src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -423,12 +423,7 @@ public override HttpRequestMessage CreateInvokeMethodRequest<TRequest>(HttpMetho
         public override async Task<HttpResponseMessage> InvokeMethodWithResponseAsync(HttpRequestMessage request, CancellationToken cancellationToken = default)
         {
             ArgumentVerifier.ThrowIfNull(request, nameof(request));
-
-            if (!this.httpEndpoint.IsBaseOf(request.RequestUri))
```

Comment:
I *think* this is intentional.  The `HttpRequestMessage` (from the `DaprClient` docs) "must be a conforming Dapr service invocation request".  This implies that the base `RequestUri` would be the Dapr sidecar host/port (as created by the `CreateInvokeMethodRequest()` method).

Reply:
For the life of me, I cannot remember my scenario for why I thought this was out of date. I'm good closing this until it comes to mind again.